packages/create-plugin/templates/common/package.json

```diff
@@ -33,16 +33,8 @@
     "@types/lodash": "^4.14.188",{{#if_eq pluginType "app"}}
     "@types/react-router-dom": "^5.3.3",{{/if_eq}}
     "@types/node": "^18.11.9",
-    "@typescript-eslint/eslint-plugin": "^5.42.1",
-    "@typescript-eslint/parser": "^5.42.1",
     "copy-webpack-plugin": "^11.0.0",
     "css-loader": "^6.7.1",
-    "eslint": "8.26.0",
```

Comment:
Nit: removing `eslint` causes a peer dependency warning from `eslint-webpack-plugin`. It's still available due to `@grafana/eslint-config`.